README.md

```diff
@@ -9,6 +9,7 @@ To add a conference to the list, please [contribute](#contributing)!
 | Conference | Date | Venue | Description |
 |------------|------|-------|-------------|
 | [JSFoo](https://jsfoo.in/2018/) | 13-14 Sep | Bengaluru | The eighth edition of India’s first JavaScript conference.
+| [India Blockchain Week](https://www.indiablockchainweek.com/) | 9 Sep | Mumbai | Blockchain Conference with Blockchain hackathon. |
```

Comment:
It's probably me being blind here :) but I don't see 9 Sep being mentioned on the website anywhere. Could you point me to a link where it says that?

Also, please add the complete duration `9 Sep - XX Sep`, since I'm guessing it's a weeklong event?